client/ratings_reviews/helpers.js

```diff
@@ -148,7 +151,16 @@ const formatReviewTile = (summary, body, photos) => {
   if (photos.length !== 0) {
     showPhotos = true;
   }
-  return [summary, body, additionalBody, showAdditionalBodyButton, showPhotos];
+  var currentHelpfulReviews = JSON.parse(sessionStorage.getItem('helpfulReviews'));
```

Comment:
Same question here with the session storage (not sure what the biz requirements spell out). I'm also wondering - should every user who visits the page see the same "count" for how many people thought the review was helpful? Because the client-side storage methods I think work only per user - so for example, I will only see the helpful review "count" on the ones I click on. Other people won't since that data is stored in my browser. Not sure if we'll need server-side storage for this data!

Reply:
Hi @jabrake, this is a great point! I believe (and I hope) that other users should be able to see the more updated helpful/report status as well because before we get to this helper, I actually make an AJAX request (when the helpful/report button was first clicked) to the API to update the helpful/report status for that particular review. So if my understanding is correct, the changes do not only happen locally but also to the API directly as well.